boot/zephyr/main.c

```diff
@@ -163,6 +182,84 @@ struct arm_vector_table {
     uint32_t reset;
 };
 
+static void __ramfunc jump_in(uint32_t reset)
```

Comment:
Can we have better name for this?

Reply:
Yes, however this is the hardest part. Any suggestions?

Reply:
Have you seen names I have invented?